integration-tests/utils/integration-utils.js

```diff
@@ -45,14 +45,16 @@ const getAnimatedDrawer = textContains => {
   return `//*[@role="dialog" and @aria-hidden="false"][contains(normalize-space(.), "${textContains}") or contains(@aria-label,"${textContains}") or @aria-labelledby=//*[contains(normalize-space(.),"${textContains}")]]`
 }
 
-const clickable = ({ text, textContains, isDescendant = false }) => {
-  const base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])`
+const clickable = ({ text, textContains, isDescendant = false, isDisabled = undefined }) => {
+  let base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])`
```

Comment:
How do you feel about this?

```suggestion
  const checkDisabled = isDisabled !== undefined ? `[@aria-disabled=${isDisabled}]` : ''
  const base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])${checkDisabled}`
```